include/grpc++/fixed_size_thread_pool.h

```diff
@@ -45,10 +45,10 @@
 
 namespace grpc {
 
-class ThreadPool GRPC_FINAL : public ThreadPoolInterface {
+class FixedSizeThreadPool GRPC_FINAL : public ThreadPoolInterface {
  public:
-  explicit ThreadPool(int num_threads);
-  ~ThreadPool();
+  explicit FixedSizeThreadPool(int num_threads);
+  ~FixedSizeThreadPool();
 
   void ScheduleCallback(const std::function<void()>& callback) GRPC_OVERRIDE;
```

Comment:
What do you think of renaming ScheduleCallback() to Add(), to be consistent with other threadpools at google?
I know this is not touched in this PR, but now that we are making this into a public API, it might be a good time to clean up names ..